File: Hire-Quest(Backend)/java/com/alok91340/gethired/dto/MeetingDto.java
/**
 * 
 */
package com.alok91340.gethired.dto;


import java.time.LocalDate;
import java.time.LocalTime;

import lombok.Data;

/**
 * @author aloksingh
 *
 */
@Data
public class MeetingDto {

private long id;
	
	private String user;
	private String hr;
	private String time;
	private String date;
	private String link;
	private boolean isAttended;
}

File: HireQuest(Frontend)/app/src/main/java/com/example/gethired/Room/MeetingDto.kt
package com.example.gethired.Room

import androidx.room.Entity
import androidx.room.PrimaryKey
import io.reactivex.rxjava3.annotations.NonNull
import org.jetbrains.annotations.NotNull
import java.time.LocalDateTime


@Entity(tableName = "meeting_table")
data class MeetingDto (
    @PrimaryKey(autoGenerate = false)
    val meetingId:Int,
    val user:String,
    val hr:String,
    val date:String,
    val time:String,
    val link:String,
    val isAttended:Boolean,
    val isReminderOn:Boolean
)
